src/grid/TSDFVolumetricMap.hpp

```diff
@@ -58,22 +57,17 @@ class TSDFVolumetricMap : public VoxelGridMap<TSDFPatch>
                     VoxelGridMap<VoxelCellType>(num_cells, resolution), truncation(truncation), min_variance(min_varaince) {}
     virtual ~TSDFVolumetricMap() {}
 
-    void mergePointCloud(const PointCloud& pc, const base::Transform3d& pc2grid, double measurement_variance = 0.01);
-    void mergePointCloud(const PointCloud& pc, const base::TransformWithCovariance& pc2grid, double measurement_variance = 0.01);
-
-    template<int _MatrixOptions>
-    void mergePointCloud(const std::vector< Eigen::Matrix<double, 3, 1, _MatrixOptions> >& pc, const base::TransformWithCovariance& pc2grid,
-                         const base::Vector3d& sensor_origin_in_pc = base::Vector3d::Zero(), double measurement_variance = 0.01);
```

Comment:
This method is used somewhere, where the covariance of `pc2grid` is zero, but the `sensor_origin_in_pc` is actually used.